client/ayon_core/plugins/load/export_otio.py

```diff
@@ -0,0 +1,231 @@
+                    )
+                else:
+                    widget = QtWidgets.QLabel("x")
+
+                layout.addWidget(widget)
+
+            input_layout.addWidget(
+                group_box, row, 1, 1, len(all_representation_names)
+            )
+
+            row += 1
+
+        export_widget = QtWidgets.QWidget()
+        export_layout = QtWidgets.QVBoxLayout(export_widget)
+
+        widget = QtWidgets.QWidget()
+        layout = QtWidgets.QHBoxLayout(widget)
+        layout.addWidget(QtWidgets.QLabel("Output Path:"))
+        self.lineedit_output_path = QtWidgets.QLineEdit()
+        layout.addWidget(self.lineedit_output_path)
+        export_layout.addWidget(widget)
+
+        self.checkbox_inspect_otio_view = QtWidgets.QCheckBox(
+            "Inspect with OTIO view"
+        )
+        export_layout.addWidget(self.checkbox_inspect_otio_view)
+
+        self.btn_export = QtWidgets.QPushButton("Export")
+        export_layout.addWidget(self.btn_export)
+
+        layout = QtWidgets.QVBoxLayout(self)
+        layout.addWidget(input_widget)
+        layout.addWidget(export_widget)
+
+        self.btn_export.clicked.connect(self.export)
+
+    def toggle_all(self):
+        representation_name = self.sender().text()
+        for item in self._representation_widgets[representation_name]:
+            item["widget"].setChecked(True)
+
+    def export(self):
+        representations = []
+        for name, items in self._representation_widgets.items():
+            for item in items:
+                if item["widget"].isChecked():
+                    representations.append(item["representation"])
+
+        anatomy = Anatomy(self._project_name)
+        clips_data = {}
+        for representation in representations:
+            version = self._version_by_representation_id[representation["_id"]]
+            name = "{}/{}".format(
+                self._version_path_by_id[version["_id"]],
+                representation["name"]
+            )
+            clips_data[name] = {
+                "path": get_representation_path_with_anatomy(
+                    representation, anatomy
+                ),
+                "frames": (
+                    version["data"]["frameEnd"] -
+                    version["data"]["frameStart"]
+                ),
+                "framerate": version["data"]["fps"]
+            }
+
+        output_path = self.lineedit_output_path.text()
+        self.export_otio(clips_data, output_path)
+
+        check_state = self.checkbox_inspect_otio_view.checkState()
+        if check_state == QtCore.Qt.CheckState.Checked:
+            run_subprocess(["otioview", output_path])
+
+    def create_clip(self, path, name, frames, framerate):
+        range = self.otio.opentime.TimeRange(
+            start_time=self.otio.opentime.RationalTime(0, framerate),
+            duration=self.otio.opentime.RationalTime(frames, framerate)
+        )
+
+        media_reference = self.otio.schema.ExternalReference(
```

Comment:
we should differenciate between sequence media source type and video files. Here is example in their code https://github.com/AcademySoftwareFoundation/OpenTimelineIO/blob/main/src/py-opentimelineio/opentimelineio/schema/image_sequence_reference.py

Reply:
I have tried to import it into Resolve as OTIO and it could not find the clips by urls. Perhaps because those were image sequences.

Reply:
Using the `ExternalReference` in Resolve works here with OTIO, but I agree we need to use the `ImageSequence` instead. Will update and we can test Resolve and Hiero again.

Reply:
@jakubjezek001 could you maybe post the steps you take to import into Resolve and Hiero?

I've tried using `ImageSequence` in Resolve, but the native OTIO implementation does not seem to support it. Again I got the correct import with `ExternalReference`.
I cant get Hiero to import with either `ExternalReference` or `ImageSequence`.

This does maybe seem like we are better off not relying on native implementations in DCCs and just parse the OTIO files ourselves, if there are going to be different implementations between DCCs.

Reply:
> This does maybe seem like we are better off not relying on native implementations in DCCs and just parse the OTIO files ourselves, if there are going to be different implementations between DCCs.

FYI, If there are differences I feel time is better spent reporting these as bugs to each individual DCC. We might want a workaround temporarily but we should definitely help push to make sure their implementations start aligning over time if they do not. The developers of those apps are likely unaware of the issues.

Reply:
@BigRoy good point!

Reply:
I was just testing the export/import with use of `ImageSequenceReference` for sequences and all was working as expected. The test had been done with Hiero 15 and DaVinci Resolve 16.5. 

@tokejepsen I am not sure what versions you were testing [here ](https://github.com/ynput/ayon-core/pull/44/files#r1488373642) it in but this is perfectly supported.

Please lets implement it with `ImageSequenceReference`.